projects/plugins/crm/admin/settings/general.page.php

```diff
@@ -319,6 +323,16 @@
 
 			<tbody>
 
+				<tr>
+					<td class="wfieldname"><label for="wpzbscrm_showfullwidthforlisting"><?php esc_html_e( 'Show listing pages in full width', 'zero-bs-crm' ); ?>:</label><br /><?php esc_html_e( 'Untick to limit the width of the listing pages', 'zero-bs-crm' ); ?></td>
+					<td style="width:540px"><input type="checkbox" class="winput form-control" name="wpzbscrm_showfullwidthforlisting" id="wpzbscrm_showfullwidthforlisting" value="1"
+					<?php
+					if ( isset( $settings['showfullwidthforlisting'] ) && $settings['showfullwidthforlisting'] === 1 ) {
+						echo ' checked="checked"';}
```

Comment:
(wondering) We haven't had any linter complaints here because of the direct output?

Reply:
No linter issues.

Reply:
Cool